addons/modules/functions/fnc_modulePoliceStation.sqf

```diff
@@ -0,0 +1,20 @@
+#include "script_component.hpp"
+/*
+ * Author: 3Mydlo3
+ * Function adds police station module to police stations list.
+ *
+ * Arguments:
+ * 0: Police station module <LOGIC>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [_module] call afsk_modules_fnc_modulePoliceStation
```

Comment:
It's a matter or preference but I tend to remove examples for not public functions, especially if they are called only from event handlers etc.

Reply:
I will remove it since it does not make any sense anyway.